web_client/js/itemList.js

```diff
@@ -1,4 +1,30 @@
 girder.wrap(girder.views.ItemListWidget, 'render', function (render) {
+    /* Chrome limits the number of connections to a single domain, which means
+     * that time-consuming requests for thumbnails can bind-up the web browser.
+     * To avoid this, limit the maximum number of thumbnails that are requested
+     * at a time.  At this time (2016-09-27), Chrome's limit is 6 connections;
+     * to preserve some overhead, use a number a few lower than that. */
+    var maxSimultaneous = 3;
+
+    /**
+     * When we might need to load another image, check how many are waiting or
+     * currently being loaded, and ask an appropriate additional number to
+     * load.
+     *
+     * @param {jquery element} parent parent under which the large_image
+     *      thumbnails are located.
+     */
+    function _loadMoreImages(parent) {
+        var loading = $('.large_image_thumbnail img.loading', parent).length;
+        if (maxSimultaneous > loading) {
+          $('.large_image_thumbnail img.waiting', parent).slice(0, maxSimultaneous - loading).each(function () {
+            var img = $(this);
+            img.removeClass('waiting').addClass('loading');
+            img.attr('src', img.attr('deferred-src'));
+          });
```

Comment:
I've fixed this indentation.  I'll make an issue to move to Girder's eslint style, as I don't see any reason not to.